sgid/state-only/index.md

```diff
@@ -10,6 +10,10 @@ title: How to Connect to the SGID via ArcSDE
 layout: page_nocrumb
 categories: []
 ---
+
+**Heads up!** As hinted at [earlier this year]({% link _posts/2020-05-04-introducing-open-sgid.md %}), we will be replacing the state-only SGID10 Enterprise Geodatabase with the [Open SGID]({% link sgid/open-sgid/index.md %}) on **December 31st, 2020**. After this date, the SGID10 database will no longer be accessible. The Open SGID contains the same datasets and is available to anyone with an internet connection. Please take time now to update your data references.
```

Comment:
**nitpick**: 

> The Open SGID contains the same datasets [as the sgid10]

This isn't accurate as we [removed or shelved](https://docs.google.com/spreadsheets/d/1FNCvSQxgQbFG--k3ZapscsTapbrxtSGlKqMcMLCsTYs/edit#gid=0) quite a few layers. I think that should be rephrased

Reply:
Makes sense. How does d0758ce25e9b26a9580592bd4bc68ab7dd000044 look to you?